components/expander.jsx

```diff
@@ -1,12 +1,15 @@
 var React = require('react');
 
+var AnchorManagerMixin = require('../mixins/anchor-manager');
+
 var Expander = React.createClass({
```

Comment:
Done! Note that `anchorId` isn't in it, because React automatically merges the `propTypes` of mixins and complains if there are any conflicts.